rebuild-docs

```diff
@@ -12,6 +12,13 @@ rm -rf ${core_dir} && git clone https://github.com/canonicalltd/ubuntu-core-docs
 rm -rf ${phone_dir} && git clone https://github.com/ubuntudesign/phone-docs.git ${phone_dir} || true
 #git clone https://github.com/canonicalltd/snapcraft-docs.git ${snapcraft_dir}
 
+# Extra build steps for Ubuntu Core docs
+(
+  cd ${core_dir}
+  repo init -u git@github.com:CanonicalLtd/ubuntu-core-docs.git
```

Comment:
I deliberately removed the installing commands from here. I think if anything it should be in the README.md, but I wasn't going to block the merging of this PR on that requirement - as for now the most important thing is that we make sure the repo command is available on the build server. I'll file an issue now about updating the README